python/cudf/cudf/tests/test_binops.py

```diff
@@ -940,7 +931,43 @@ def test_floordiv_zero_float64(series_dtype, divisor_dtype, scalar_divisor):
     else:
         pd_div = pd.Series([0], dtype=divisor_dtype)
         cudf_div = cudf.from_pandas(pd_div)
-    utils.assert_eq((sr // pd_div), (cr // cudf_div))
+    utils.assert_eq(sr // pd_div, cr // cudf_div)
+
+
+@pytest.mark.parametrize(
+    "scalar_divisor",
+    [
+        pytest.param(
+            False,
+            marks=pytest.mark.xfail(
+                reason="https://github.com/rapidsai/cudf/issues/12162"
+            ),
+        ),
+        pytest.param(
+            True,
+            marks=pytest.mark.xfail(
+                reason="https://github.com/rapidsai/cudf/issues/12162"
+            ),
+        ),
+    ],
+)
+def test_floordiv_zero_bool(scalar_divisor):
```

Comment:
Rather than marking each parameter, why not just mark the test so that you can parametrize with [False, True]?

Reply:
Good point; done.